scripts/check_connectivity.py

```
@@ -0,0 +1,50 @@
+async def check_connectivity():
+    settings = Settings()
+    print(f"Checking connectivity for storage_backend={settings.storage_backend}...")
+
+    # 1. PostgreSQL
+    try:
+        print(f"Connecting to PostgreSQL at {settings.postgres_host}...")
+        postgres = await PostgresStorageAdapter.create(settings)
+        stats = await postgres.list_projects()
+        print(f"✅ PostgreSQL connected! Projects: {stats}")
+        await postgres.dispose()
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)